textproc/libxmlxx5/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+compiler.cxx_standard 2017
+
+name                libxmlxx5
+set gname           libxml++
+version             5.0.1
+set branch          [join [lrange [split ${version} .] 0 1] .]
+categories          textproc
+license             LGPL
```

Comment:
according to [upstream](https://libxmlplusplus.github.io/libxmlplusplus/#license) it's `LGPL-2+`